src/store/Business/getters.ts

```diff
@@ -0,0 +1,161 @@
+import { CorpTypeCd, EntityState } from '@/enums'
+import { BusinessIF, BusinessWarningIF } from '@/interfaces'
+import { DateUtilities } from '@/services/'
+
+export default {
+  /** The business number (aka Tax ID). */
+  getBusinessNumber (state: BusinessIF): string {
+    return state.taxId
+  },
+
+  /** The business warnings list. */
+  getBusinessWarnings (state: BusinessIF): Array<BusinessWarningIF> {
+    return state.warnings
+  },
+
+  /** The business state. */
+  getBusinessState (state: BusinessIF): EntityState {
+    return state.state
+  },
+
+  /** The founding date. */
+  getFoundingDate (state: BusinessIF): Date {
+    return DateUtilities.apiToDate(state.foundingDate)
```

Comment:
This is significant: we put some of the logic in the getters. In this example, we return something "different" from the business info object in the store.